main.py

```diff
@@ -2,6 +2,40 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+    actors = [
```

Comment:
The checklist said to add values to the list of tuples, but this option is also possible